client/v3/snapshot/v3_snapshot.go

```diff
@@ -95,8 +96,8 @@ func SaveWithVersion(ctx context.Context, lg *zap.Logger, cfg clientv3.Config, d
 	)
```

Comment:
maybe we use `(_ string, err error)` for return value and update L94 with `resp.Version`.
In go, it's unlikely to use `resp.Version` if the error is not nil. So, I think we can update the L79/L82/L88/L98 with `return "", error`. What do you think?